auto-increment: Mention `FORCE` (#7502)
pingcap/docs: auto-increment.md

- It cannot be specified on the same column with the `DEFAULT` column value.
- `ALTER TABLE` cannot be used to add the `AUTO_INCREMENT` attribute.
- `ALTER TABLE` can be used to remove the `AUTO_INCREMENT` attribute. However, starting from v2.1.18 and v3.0.4, TiDB uses the session variable `@@tidb_allow_remove_auto_inc` to control whether `ALTER TABLE MODIFY` or `ALTER TABLE CHANGE` can be used to remove the `AUTO_INCREMENT` attribute of a column. By default, you cannot use `ALTER TABLE MODIFY` or `ALTER TABLE CHANGE` to remove the `AUTO_INCREMENT` attribute.

> - `ALTER TABLE` requires the `FORCE` option to set the `AUTO_INCREMENT` value to a lower value.

Reviewer: ```suggestion
- `ALTER TABLE` requires the `FORCE` option to set the `AUTO_INCREMENT` value to a smaller value.
```

> - Setting the `AUTO_INCREMENT` to a value less than `MAX(<auto_increment_column>)` leads to duplicate keys as pre-existing values are not skipped.

Reviewer: ```suggestion
- Setting the `AUTO_INCREMENT` to a value smaller than `MAX(<auto_increment_column>)` leads to duplicate keys because pre-existing values are not skipped.
```
